File: C/Bubble_Sort.c
//Implementation of Bubble Sort
#include<stdio.h>
void main(){
    int i, j, a[40], temp, n;
    printf("How many numbers do you want to sort = ");
    scanf("%d", &n);
    printf("Enter you numbers: \n");
    for(i = 0; i < n; i++){
        printf("Number %d : ", i + 1);
        scanf("%d", &a[i]);
    }

    //Bubble sort

    for(i = 1; i < n; i++){
        for(j = 0; j < n - i; j++){
            if(a[j] > a[j + 1]){
                temp = a[j];
                a[j] = a[j + 1];
                a[j + 1] = temp;
            }
        }
    }

    printf("\nSorted Array in ascending order:\n");
    for(i = 0; i < n; i++){
        printf("Number %d : ", i + 1);
        printf("%d\n", a[i]);
    }

    for(i = 1; i < n; i++){
        for(j = 0; j < n - i; j++){
            if(a[j] < a[j + 1]){
                temp = a[j];
                a[j] = a[j + 1];
                a[j + 1] = temp;
            }
        }
    }

    printf("\nSorted Array in descending order:\n");
    for(i = 0; i < n; i++){
        printf("Number %d : ", i + 1);
        printf("%d\n", a[i]);
    }
}

File: python/bubble_sort.py
print("Enter the number of values you want to sort: ")
list1 = []
n = int(input())
for index in range(n):
    print("Number " + str(index + 1) + ' : ', end='')
    list1.append(int(input()))

for base_counter in range(1, n):
    for counter in range(0, n - base_counter):
        if(list1[counter] > list1[counter + 1]):
            list1[counter], list1[counter + 1] = list1[counter + 1], list1[counter]

for index in range(n):
    print("Number " + str(index + 1) + ": " + str(list1[index]))
